test/studies/1brc/aggregators.good

```diff
@@ -0,0 +1,415 @@
+aggregators.chpl:86: In method 'flush':
+aggregators.chpl:88: warning: borrowed sharedLock(map(bytes,tempData,false)) is being used in a 'manage' statement via its 'enterContext' and 'exitContext' methods. However, borrowed sharedLock(map(bytes,tempData,false)) does not implement contextManager. In the future, this will result in an error.
```

Comment:
This seems a bit concerning.

Reply:
This is a bug related to how that warning is generated. See: https://github.com/chapel-lang/chapel/issues/24408